app/components/trainees/confirmation/personal_details/view.rb

```diff
@@ -0,0 +1,44 @@
+module Trainees
+  module Confirmation
+    module PersonalDetails
+      class View < GovukComponent::Base
+        attr_accessor :trainee
+
+        def initialize(trainee:)
+          @trainee = trainee
+        end
+
+        def full_name
+          return I18n.t("confirmation.not_entered") unless trainee.first_names
+
+          "#{trainee.first_names} #{trainee.last_name}".upcase
```

Comment:
```suggestion
          "#{trainee.first_names.capitalize} #{trainee.last_name.capitalize}".upcase
```

Actually, I think we need to pull in `middle_names` as well. It might be worth checking if values are present before calling `capitalize`.